packages/paste-website/src/components/site-wrapper/SiteHeader.tsx

```diff
@@ -29,34 +42,64 @@ export const SiteHeader: React.FC = () => {
   return (
     <Box
       as="aside"
-      backgroundColor="colorBackgroundBody"
-      borderColor="colorBorderLight"
-      borderStyle="none none solid none"
-      borderBottomWidth="borderWidth10"
-      paddingLeft="space200"
-      paddingRight="space80"
-      paddingTop="space50"
-      paddingBottom="space50"
+      display="flex"
+      alignItems="center"
+      backgroundColor="colorBackgroundPrimaryDarkest"
```

Comment:
Shouldn't this be using an inverse background token?

Reply:
Also found a bug in inverse anchor. Hover state doesn't pass contrast 

![image](https://user-images.githubusercontent.com/368249/95142758-48b7f100-0729-11eb-8a9f-9f46451085b0.png)

Reply:
I'll create a ticket it the backlog for this.

Reply:
Following this design for the background color: https://app.abstract.com/projects/39ed9aa7-cca8-4e4f-8f16-d8f9d36d251a/branches/master/commits/latest/files/6a521d86-90ac-4db8-83b6-9dbfd0c84e7f/layers/B33BCCE8-FABA-4C0D-A0CF-3BCC8986465A?collectionId=bc3992f2-7d22-4394-967d-a62ba262e26f&collectionLayerId=efe4c114-c1ec-47a1-a35d-8d2f2ad50c78

Reply:
Yeah, `colorBackgroundPrimaryDarkest` is the right token. It's not our inverse background color token, which is the darkest color on the gray scale. This is the darkest color on the blue / primary scale. We discussed naming of this token a lottttt and landed on this, since we don't intend for this to be the inverse background color in most cases.